stencil/src/components/pulumi-api-doc-filterable-nav/pulumi-api-doc-filterable-nav.scss

```diff
@@ -14,7 +14,7 @@
 
     .navigation-filter-input {
         height: 20px;
-        width: 85%;
+        width: 90%;
```

Comment:
That's to account for the space needed next to the scroll bar.  Definitely could accomplish this differently, by accounting for the input padding and then setting this to 100%, could go that way if you think it would give more clarity.